Cargo.toml

```diff
@@ -15,41 +14,39 @@ edition = "2021"
 rust-version = "1.56"
 
 [features]
-
 default = ["compute", "image", "network", "native-tls", "object-storage"]
 compute = []
 image = []
 network = []
 native-tls = ["reqwest/default-tls", "osauth/native-tls"]
-object-storage = []
+object-storage = ["tokio-util"]
 rustls = ["reqwest/rustls-tls", "osauth/rustls"]
 
 [dependencies]
-
-bytes = { version = "^0.5" }
+async-trait = "^0.1"
 chrono = { version = "^0.4", features = ["serde"] }
 eui48 = { version = "^1.0", features = ["disp_hexstring", "serde"] }
-fallible-iterator = "^0.2.0"
 ipnet = { version = "^2.0", features = ["serde"] }
 futures = "^0.3"
 log = "^0.4"
-osauth = { version = "^0.3.3", default-features = false }
-osproto = "^0.2.1"
+osauth = { version = "^0.4", default-features = false, features = ["stream"] }
+osproto = "^0.2.0"
```

Comment:
I wonder why the downgrade? On top of that, osproto is deprecated, I wonder if we still need it here.

Reply:
The downgrade seems to have come due to the fact that this branch was originally based on the old `async` branch and then rebased onto master later, and I may have missed this when solving the resulting conflicts.

You're right that `osauth` does export all the types that `osproto` provided, so I changed the imports and removed `osproto` as a dependency.